packages/paste-website/src/component-examples/VisualPickerExamples.ts

```diff
@@ -0,0 +1,828 @@
+              </MediaObject>
+              <Box display="flex" columnGap="space50">
+                <StatusBadge as="span" variant="ConnectivityOffline">
+                  Offline
+                </StatusBadge>
+                <AvatarGroup size="sizeIcon20" variant="user">
+                  <Avatar name="site user1" icon={CommunityIcon} />
+                  <Avatar name="site user2" icon={CommunityIcon} />
+                </AvatarGroup>
+              </Box>
+            </Box>
+          </VisualPickerRadio>
+          <VisualPickerRadio id={useUID()} value='3' labelText="ACME (website)">
+            <Box display="flex" justifyContent="space-between" alignItems="center">
+              <MediaObject verticalAlign="center">
+                <MediaFigure spacing="space50">
+                  <Avatar variant="entity" icon={CodeIcon} size="sizeIcon90" name="code" color="decorative20" />
+                </MediaFigure>
+                <MediaBody>
+                  <Text as="div" fontWeight="fontWeightSemibold">
+                    ACME
+                  </Text>
+                  <Text as="div" color="colorTextWeak">
+                    Website
+                  </Text>
+                </MediaBody>
+              </MediaObject>
+              <Box display="flex" columnGap="space50">
+                <StatusBadge as="span" variant="ConnectivityAvailable">
+                  Enabled
+                </StatusBadge>
+                <AvatarGroup size="sizeIcon20" variant="user">
+                  <Avatar name="site user1" icon={CommunityIcon} />
+                  <Avatar name="site user2" icon={CommunityIcon} />
+                  <Avatar name="site user3" icon={CommunityIcon} />
+                </AvatarGroup>
+              </Box>
+            </Box>
+          </VisualPickerRadio>
+        </VisualPickerRadioGroup>
+    );
+}
+render(
+    <VisualPickerExample/>
+)
+`.trim();
+
+export const VisualPickerRadioHorizontal = `
+const VisualPickerExample = () => {
+    const [value, setValue] = React.useState('')
+    return (
+        <VisualPickerRadioGroup
+          orientation="horizontal"
+          name={useUID()}
+          id={useUID()} 
+          legend="Select a plan"
+          onChange={(newValue) => {
+            setValue(newValue);
+          }}
+          helpText="The new plan will take effect at the start of the upcoming cycle."
+          value={value}
+        >
+          <VisualPickerRadio id={useUID()} value='1' labelText="30 US dollars per user per month">
+            <Box display="flex" flexDirection="column" rowGap="space30">
+              <Box display="flex" flexDirection="column" rowGap="space30">
+                <Text as="span" fontWeight="fontWeightSemibold" fontSize="fontSize60">
+                  Basic plan
+                </Text>
+                <Text as="span" fontWeight="fontWeightNormal" fontSize="fontSize30">
+                  Most basic option
+                </Text>
```

Comment:
chore: set the second line of text to colorTextWeak, along with the other items.